middle_end/flambda2/simplify/env/downwards_acc.ml

```diff
@@ -53,10 +56,12 @@ let [@ocamlformat "disable"] print ppf
     Data_flow.print data_flow
     Continuation.Set.print demoted_exn_handlers
     Code_id.Set.print code_ids_to_remember
+    (Or_unknown.print Closure_offsets.print) closure_offsets
 
-let create denv continuation_uses_env =
+let create denv continuation_uses_env closure_offsets =
```

Comment:
Would you mind updating this to match `Upwards_acc` (using a `compute_closure_offsets` argument) ?

Reply:
done